File: legacy/android/java/StorylyDemo/app/src/main/java/com/appsamurai/storylydemo/CustomThemeActivity.java
package com.appsamurai.storylydemo;

import android.os.Bundle;

import androidx.appcompat.app.AppCompatActivity;

import com.appsamurai.storyly.StorylyInit;
import com.appsamurai.storyly.StorylyView;

public class CustomThemeActivity extends AppCompatActivity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_custom_theme);

        StorylyView storyly_view_default = findViewById(R.id.storyly_view_default);
        storyly_view_default.setStorylyInit(new StorylyInit(Tokens.STORYLY_INSTANCE_TOKEN));


        StorylyView storyly_custom_theme = findViewById(R.id.storyly_custom_theme);
        storyly_custom_theme.setStorylyInit(new StorylyInit(Tokens.STORYLY_INSTANCE_TOKEN));

        // storyly_custom_theme.setStoryGroupTextColor(Color.parseColor("#f03932"));

        // storyly_custom_theme.setStoryGroupIconBackgroundColor(Color.parseColor("#4CAF50"));

        // Integer[] seenBorder = {Color.parseColor("#C5ACA5"), Color.parseColor("#000000")};
        // storyly_custom_theme.setStoryGroupIconBorderColorSeen(seenBorder);

        // Integer[] notSeenBorder = {Color.parseColor("#FB3200"), Color.parseColor("#FFEB3B")};
        // storyly_custom_theme.setStoryGroupIconBorderColorNotSeen(notSeenBorder);

        // storyly_custom_theme.setStoryGroupPinIconColor(Color.parseColor("#3F51B5"));

        // storyly_custom_theme.setStoryItemTextColor(Color.parseColor("#FF0057"));

        // Integer[] borderColor = {Color.parseColor("#FB3200"), Color.parseColor("#FFEB3B")};
        // storyly_custom_theme.setStoryItemIconBorderColor(borderColor);

        // Integer[] progressBar = {Color.parseColor("#FB3200"), Color.parseColor("#FFEB3B")};
        // storyly_custom_theme.setStoryItemProgressBarColor(progressBar);
    }
}

File: android/kotlin/StorylyDemo/app/src/main/java/com/appsamurai/storylydemo/CustomThemeActivity.kt
package com.appsamurai.storylydemo

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.appsamurai.storyly.StorylyInit
import com.appsamurai.storylydemo.databinding.ActivityCustomThemeBinding

class CustomThemeActivity : AppCompatActivity() {
    private lateinit var binding: ActivityCustomThemeBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityCustomThemeBinding.inflate(layoutInflater)
        setContentView(binding.root)

        binding.storylyViewDefault.storylyInit = StorylyInit(STORYLY_INSTANCE_TOKEN)

        binding.storylyCustomTheme.storylyInit = StorylyInit(STORYLY_INSTANCE_TOKEN)

        // storyly_custom_theme.setStoryGroupTextColor(Color.parseColor("#f03932"))

        // storyly_custom_theme.setStoryGroupIconBackgroundColor( Color.parseColor("#4CAF50"))

        // val seenBorder = arrayOf(Color.parseColor("#C5ACA5"), Color.parseColor("#000000"))
        // storyly_custom_theme.setStoryGroupIconBorderColorSeen(seenBorder)

        // val notSeenBorder = arrayOf(Color.parseColor("#FB3200"), Color.parseColor("#FFEB3B"))
        // storyly_custom_theme.setStoryGroupIconBorderColorNotSeen(notSeenBorder)

        // storyly_custom_theme.setStoryGroupPinIconColor(Color.parseColor("#3F51B5"))

        // storyly_custom_theme.setStoryItemTextColor(Color.parseColor("#FF0057"))

        // val borderColor = arrayOf(Color.parseColor("#FB3200"), Color.parseColor("#FFEB3B"))
        // storyly_custom_theme.setStoryItemIconBorderColor(borderColor)

        // val progressBar = arrayOf(Color.parseColor("#FB3200"), Color.parseColor("#FFEB3B"))
        // storyly_custom_theme.setStoryItemProgressBarColor(progressBar)
    }
}